packages/mobile/src/screens/profile-screen/ArtistRecommendations/ArtistRecommendations.tsx

```diff
@@ -102,7 +102,7 @@ export const ArtistRecommendations = (props: ArtistRecommendationsProps) => {
 
   const [idsToFollow, setIdsToFollow] = useState<ID[] | null>(null)
   const artistsToFollow = useSelector<CommonState, User[]>((state) =>
-    Object.values(getUsers(state, { ids: idsToFollow }))
+    Object.values(getUsers(state, { ids: idsToFollow || [] }))
```

Comment:
this was causing us to fetch the entire cache because it defaulted to null

Reply:
Wow crazy

Reply:
Maybe best to handle on the app definition side?

Reply:
i guess yeah we could update the cache selectors to return nothing when falsey values are passed in. not sure if there are other consequences but will look into it. seems better

Reply:
I'm actually way more nervous about that change, but feel that this is safe to cherry pick, so going to do this for now and investigate that separately. we have some logic that intentionally calls getAllEntries there https://github.com/AudiusProject/audius-protocol/blob/f5672369c2ecf8692508a0ccdb0b56118473c758/packages/common/src/store/cache/users/selectors.ts#L66